pkg/storage/client.go

```diff
@@ -94,15 +94,19 @@ func (c *Client) Commit(ctx context.Context, owner string, name string, rev stri
 	}
 
 	return Commit{
-		Hash:           res.GetHash(),
-		Tree:           res.GetTree(),
-		Parent:         res.GetParent(),
-		Message:        res.GetMessage(),
-		Author:         res.GetAuthor(),
-		AuthorEmail:    res.GetAuthorEmail(),
-		AuthorDate:     time.Unix(res.GetAuthorDate(), 0),
-		Committer:      res.GetCommitter(),
-		CommitterEmail: res.GetCommitterEmail(),
-		CommitterDate:  time.Unix(res.GetCommitterDate(), 0),
+		Hash:    res.GetHash(),
+		Tree:    res.GetTree(),
+		Parent:  res.GetParent(),
+		Message: res.GetMessage(),
+		Author: Author{
```

Comment:
I remember that I've actually put these strings inline for simplicity. IMHO there's no real need to specifically create a Author struct. I see Author and Committer pretty much scoped to this.
Flattening this seems totally reasonable to me here.